tools/hangar/__snapshots__/test_corpus/api.w_compile_tf-aws.md

```diff
@@ -8,7 +8,7 @@ module.exports = function({ counter }) {
     }
     async handle(request)  {
       {
-        const count = (await counter.inc());
```

Comment:
Because we added "skip" to this test, the snapshot wasn't updated for around a month, so there's a lot of unrelated changes here